ui/hooks/useTokensToSearch.js

```diff
@@ -9,6 +9,8 @@ import {
   getCurrentCurrency,
```

Comment:
I remember you told me you were encountering some problem with this file. What was it again?

Reply:
The application was crashing when I add shuffling to the tokenList Data, I will leave a comment at the line of code.